src/org/thoughtcrime/securesms/ConversationActivity.java

```diff
@@ -157,6 +158,7 @@ public class ConversationActivity extends PassphraseRequiredActionBarActivity
 {
   private static final String TAG = ConversationActivity.class.getSimpleName();
 
+  public static final String ACCOUNT_ID_EXTRA        = "account_id";
```

Comment:
it can't be in `NotificationCenter` only because then `CHAT_ID_EXTRA` would also need to be moved there and it is used by the maps activities as well not only by `NotificationCenter`, I find it fine to have the "*EXTRA" in the class that process the Intent, and when used in the classes launching the Intent they are used with the class name prefix ex. `ConversationActivity.ACCOUNT_ID_EXTRA` so it is clear what `ACCOUNT_ID_EXTRA` is referred.

OTOH, using unique values for  each one of them (perhaps using the class name as prefix) would avoid using the wrong *EXTRA

Reply:
`CHAT_ID_EXTRA` is only defined in ConversationActivity and everything else `import static`s it from there, so, let's also do this for `ACCOUNT_ID_EXTRA`?

Reply:
`CHAT_ID_EXTRA` is **not** only defined in `ConversationActivity` but in each class that receives `CHAT_ID_EXTRA` in Intent extras, ex. `MarkReadReceiver` and `RemoteReplyReceiver`

Reply:
Lol, you're right

Reply:
Sorry for the confusion